viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx

```diff
@@ -43,9 +43,9 @@ export class TraceExplorerWidget extends ReactWidget {
     @inject(TraceViewerContribution)
     protected readonly traceViewerContribution!: TraceViewerContribution;
 
-    private OPENED_TRACE_TITLE = 'Opened experiments';
+    private OPENED_TRACE_TITLE = 'Open Traces';
```

Comment:
If you're going to update this patch for the plural of analyses, maybe this should be "Opened Traces", or just "Traces"?

Reply:
Ok, I put it back to Opened (even if I have to redo the screenshot :-))

Reply:
Screenshot said already "Opened Traces"